### ISSUE TITLE
RUB-33 ⁃ Mobile - Selected Pin Popup

### ISSUE THREAD

**Issue by thatdoogieguy** [thatdoogieguy/MapMyBins#130]
When clickin on a pin show this popup.

Popup displays

* Asset Name
* Asset Type
* Asset Location
* Last image in Asset Images

Buttons across bottom:

* Edit Asset
* Show Image Strip
* Take a Photo
* Navigate to Asset
* Favourite Asset (highlight if already Favourited / toggle)
* Delete Asset

Not sure if buttons are big enough to be honest but will see how it goes.

┆Attachments: <a href="https://smartsensor.atlassian.net/secure/attachment/10415/SelectedPinPopup.png">SelectedPinPopup.png</a>

**Comment by thatdoogieguy** [comment]
➤ User  commented:

[~admin] What is the Asset Location format? It is the same for right drawer soI  will ask only once here.
1) What does it do "Show Image Strip" ?
2) What does it do  "Navigate to Asset"? 
3) I am not sure but I guess there is no preview images for a pin. So I just use street view. Also I can not find API uploading photos for a pin.

**Comment by thatdoogieguy** [comment]
➤ User  commented:

1) Show image strip should show a historical list of images for that location.
2) Navigate to asset means show directions to asset, show path as discussed if possible but low priority for now.
3) [~indr4_winata] ?

**Comment by thatdoogieguy** [comment]
➤ User  commented:

[~indr4_winata]  What is historical list of images?
